CITATION.cff

```diff
@@ -5,16 +5,45 @@ cff-version: 1.2.0
 title: deal.II
 message: >-
   If you use this software, please cite the overview paper
-  (doi) as well as the paper for the major release you used.
+  (preferred citation) as well as the paper for the major release you used.
 type: software
 authors:
   - name: The deal.II authors
     website: 'https://www.dealii.org/authors.html'
 identifiers:
-  - type: doi
-    value: 10.1016/j.camwa.2020.02.022
   - type: url
     value: 'https://dealii.org/publications.html'
 repository-code: 'https://github.com/dealii/dealii'
 url: 'https://dealii.org'
 license: LGPL-2.1-or-later
```

Comment:
Do we still need this? I don't see it showing up at https://github.com/jpthiele/dealii/tree/cff-preferred-citation in "Cite this repository".

Reply:
Also I wasn't aware that GitHub shows different information for each branch based on the files therein.
Good to know for CFF files and such!

Reply:
Yes, the DOI is also listed below in `preferred-citation`. I don't know if we need the the part above for `GitHub` to process the file properly or not.

Reply:
cffinit is happy with it and it was optional when I wrote the initial version.
What I can't figure out or seems a current GitHub implementation issue
is that the custom message is not being displayed.